server/src/internalClusterTest/java/org/opensearch/snapshots/ConcurrentSnapshotsIT.java

```diff
@@ -1033,35 +1033,35 @@ public void testStartDeleteDuringFinalizationCleanup() throws Exception {
         final String snapshotName = "snap-name";
         blockMasterFromDeletingIndexNFile(repoName);
```

Comment:
`blockMasterFromDeletingIndexNFile` -> `blockClusterManagerFromDeletingIndexNFile`

Reply:
It's a public method, and it has been published to maven, so I plan to change the name in next major version (3.0), and it's tracked in the issue https://github.com/opensearch-project/OpenSearch/issues/1684